tasks/test.yaml

```diff
@@ -70,6 +70,12 @@ tasks:
             npm ci && \
             npx playwright test \
             "
+      - description: "Run E2E Tests"
+      - cmd: |
+          for package in src/*; do
+            uds run -f ${package}/tasks.yaml e2e-test --no-progress
+          done
```

Comment:
In the case of grafana (or other playwright tests if we had them) this ends up re-running the tests. The previous lines here will run any playwright tests, is there a reason this doesn't just run `npm ci && npx jest` against all the jest tests under `e2e/test`?

With Grafana I had structured it so that this task would run all playwright tests, with the assumption that everything in core is installed. Then the task specifically in the grafana task file ran assuming only the functional layer of monitoring/dependencies was installed, so it actually tests a few less integrations. Not sure if that will be helpful/apply with any others?